src/pretix/settings.py

```diff
@@ -3,7 +3,7 @@
 import os
 import sys
 from urllib.parse import urlparse
-
+from .settings_helpers import *
```

Comment:
Please not use `import *`. Be explicit about what you want to import.
[PEP8 coding style](https://peps.python.org/pep-0008/#imports)

Reply:
I have made these changes.